src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs

```diff
@@ -1472,5 +1484,35 @@ private void ResetScrollIndicator(bool forced = false)
 			}
 		}
 		#endregion
+
+		public void ScrollToHorizontalOffset(double offset)
+		{
+			_ = ChangeView(offset, null, null, false);
+		}
+
+		public void ScrollToVerticalOffset(double offset)
+		{
+			_ = ChangeView(null, offset, null, false);
+		}
+
+		// Indicates whether ScrollViewer should ignore mouse wheel scroll events (not zoom).
+		public bool ArePointerWheelEventsIgnored { get; set; } = false;
+
+		internal bool BringIntoViewport(Rect bounds,
+			bool skipDuringTouchContact,
+			bool skipAnimationWhileRunning,
+			bool animate)
+		{
+#if __WASM__
+			return ChangeView(bounds.X, bounds.Y, null, true);
+#else
+			return ChangeView(bounds.X, bounds.Y, null, !animate);
+#endif
+		}
+
+		internal bool IsInManipulation => IsInDirectManipulation || m_isInConstantVelocityPan;
+
+		internal bool ForceChangeToCurrentView { get; set; } = false;
```

Comment:
add an xml comment explaining what this does

Reply:
Completed.